src/main/java/org/apache/tomcat/jakartaee/Migration.java

```diff
@@ -322,7 +325,7 @@ private void migrateArchiveStreaming(InputStream src, OutputStream dest) throws
                 String destName = profile.convert(srcName);
                 if (srcZipEntry.getMethod() == ZipEntry.STORED) {
                     ByteArrayOutputStream tempBuffer = new ByteArrayOutputStream((int) (srcZipEntry.getSize() * 1.05));
-                    migrateStream(srcName, srcZipStream, tempBuffer);
+                    convertedArchive = convertedArchive | migrateStream(srcName, srcZipStream, tempBuffer);
```

Comment:
This looks fishy to me: once convertedArchive is true, no more streams will be migrated.

Was that the intent?